CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.27.0
```

Comment:
I think this should have been `Unreleased` but doesn’t matter.

Reply:
Fixed in envelope PR